lib/dart.yaml

```diff
@@ -1,194 +1,209 @@
+        - avoid-long-files:
+              max-length: 500
+        - avoid-long-functions:
+              max-length: 80
+              ignored-names:
+                  - build
+        - avoid-long-parameter-list:
+              ignore-optional: true
+              max-amount: 5
+              ignored-names:
+                  - copyWith
+        - avoid-long-records:
+              max-amount: 3
+        # https://github.com/dart-lang/sdk/issues/52909
+        - avoid-map-keys-contains
+        - avoid-missed-calls
+        - avoid-missing-enum-constant-in-map
+        - avoid-missing-interpolation
+        - avoid-misused-wildcard-pattern
+        - avoid-mixing-named-and-positional-fields
+        - avoid-mutating-parameters
+        - avoid-negated-conditions
+        - avoid-nested-conditional-expressions:
+              acceptable-level: 2
+        - avoid-nested-futures
+        - avoid-nested-records
+        - avoid-nested-streams-and-futures
+        - avoid-nested-switch-expressions
+        - avoid-nested-switches
+        - avoid-non-null-assertion
+        - avoid-nullable-interpolation
+        - avoid-nullable-parameters-with-default-values
+        - avoid-nullable-tostring
+        - avoid-one-field-records
+        - avoid-passing-async-when-sync-expected
+        - avoid-passing-default-values
+        - avoid-passing-self-as-argument
+        - avoid-positional-record-field-access
+        - avoid-recursive-calls
+        - avoid-redundant-async
+        - avoid-redundant-else
+        - avoid-referencing-discarded-variables
+        - avoid-self-assignment
+        - avoid-self-compare
+        - avoid-shadowed-extension-methods
+        - avoid-shadowing
+        - avoid-throw-in-catch-block
+        - avoid-throw-objects-without-tostring
+        - avoid-unassigned-late-fields
+        - avoid-unassigned-stream-subscriptions
+        - avoid-unconditional-break
+        - avoid-unnecessary-conditionals
+        - avoid-unnecessary-futures
+        - avoid-unnecessary-if
+        - avoid-unnecessary-local-late
+        - avoid-unnecessary-negations
+        - avoid-unnecessary-nullable-return-type
+        - avoid-unnecessary-return
+        - avoid-unnecessary-super
+        - avoid-unsafe-collection-methods
+        - avoid-unused-after-null-check
+        - avoid-unused-parameters
+        - binary-expression-operand-order
+        - double-literal-format:
+              explicit-trailing-zero: true
+        - format-comment:
+              only-doc-comments: false
+        - function-always-returns-null
+        - map-keys-ordering
+        - match-positional-field-names-on-assignment
+        - member-ordering:
+              order:
+                  - public-fields
+                  - private-fields
+                  - private-late-final-fields
+                  - public-getters
+                  - private-getters
+                  - public-setters
+                  - private-setters
```

Comment:
Ordering fields, getters and setters within different classes other than widgets (WM, Model, Scopes)